CollectionActions.rb

```diff
@@ -7,108 +7,110 @@
 # These actions should involve the WHOLE plate not individual wells.
 # NOTE: The collection is doing the whole action
 module CollectionActions
-  # Store all input collections from all operations
+  # Store all items used in input operations
   #
   # @param operations [OperationList] the list of operations
-  # @param location [String] the location that the items are to be moved to
+  # @param location [String] the storage location
   def store_input_collections(operations, location: nil)
-    show do
-      title 'Put Away the Following Items'
-      table table_of_job_object_location(operations, role: 'input',
-              location: location)
-    end
+    store_collection_materials(operations, location: location, role: 'input')
+    # show do
+    #   title 'Put Away the Following Items'
+    #   table material_storage_locations(operations, role: 'input',
+    #           location: location)
+    # end
   end
 
-  # Stores all output collections from all operations
+  # NOTE: Is there a reason these are seperate methods? 
+  # Stores all items used in output operations
   #
   # @param operations [OperationList] the operation list where all
   # output collections should be stored
   def store_output_collections(operations, location: nil)
-    show do
-      title 'Put Away the Following Items'
-      table table_of_job_object_location(operations, role: 'output',
-              location: location)
-    end
+    store_collection_materials(operations, location: location, role: 'output')
+    # show do
+    #   title 'Put Away the Following Items'
+     #  table material_storage_locations(operations, role: 'output',
+      #         location: location)
+   #  end
   end
 
-  # Store collections
+  # Instructions for technician on how and where to store materials
```

Comment:
I've been trying to clarify in the notes which methods are instructions for the technician to work with literal items and which are instructions for Aquarium to work with virtual items.

Reply:
Ahh thats a good idea.  In the future I will do that